src/hw-4/models/group.model.ts

```diff
@@ -0,0 +1,23 @@
+import { DataTypes } from 'sequelize';
+
+export default (sequelize: any) => {
+    return sequelize.define('group', {
+        id: {
+            type: DataTypes.UUID,
+            defaultValue: DataTypes.UUIDV4,
+            primaryKey: true,
+            allowNull: false
+        },
+        name: {
+            type: DataTypes.TEXT,
+            allowNull: false
+        },
+        permission: {
+            type: DataTypes.ARRAY(DataTypes.TEXT),
```

Comment:
`permissions`, also it has to be an array of enum values (written down in the hometask).

Reply:
nevermind, it's going to be array of text values here. I think you can use [this link](https://sequelize.org/api/v6/class/src/data-types.js~enum) to define enum type, but seems it'll be supported only in postgre. But then you'll have ORM level validation for this field.